documentation/src/main/asciidoc/architecture.asciidoc

```diff
@@ -63,16 +63,16 @@ pattern aka. atomic conversation.
 ====
 
 
-=== Back end
+=== Backend
 
-Hibernate Search offers the ability to let the batched work being processed by different back ends.
-Several back ends are provided out of the box and you have the option to plugin your own. It is
-important to understand that in this context back end encompasses more than just the configuration
-option `hibernate.search.default.worker.backend`. This property just specifies a implementation of
+Hibernate Search offers the ability to let the batched work being processed by different backends.
```

Comment:
"let the batched work being" => "let the batched work be", unless the Beatles spoke a bad English... ?